browser/src/VariantPage/VariantOccurrenceTable.tsx

```diff
@@ -43,15 +43,23 @@ const NoWrap = styled.span`
 type VariantContext = 'exome' | 'genome' | 'joint'
 const renderGnomadVariantFlag = (variant: Variant, context: VariantContext) => {
   if (!variant[context]) {
-    return <Badge level="error">No variant</Badge>
+    return context === 'joint' ? <div /> : <Badge level="error">No variant</Badge>
   }
   const { filters } = variant[context]!
 
   if (filters.length === 0) {
-    return <Badge level="success">Pass</Badge>
+    return context === 'joint' ? <div /> : <Badge level="success">Pass</Badge>
   }
 
-  return filters.map((filter: any) => <QCFilter key={filter} filter={filter} />)
+  return filters.map((filter) => {
+    let data = {}
```

Comment:
as a rule we should try and avoid `let` if it can be avoided without making the code too convoluted. looks like you could rewrite this neatly like so:

```
return filters.map((filter) => {
  const data = filter === 'discrepant_frequencies' ? {cochran_p_value: blah blah} : {}
  return <QCFilter key={filter} filter={filter} data={data} />
})
```
```

Reply:
Aha, assigning a variable with a ternary here is pretty neat!